Formula/opencolorio.rb

```diff
@@ -18,21 +19,18 @@ class Opencolorio < Formula
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "little-cms2"
-  depends_on "python@3.9"
+  depends_on "python@3.10"
 
   def install
-    args = std_cmake_args + %W[
-      -DCMAKE_VERBOSE_MAKEFILE=OFF
+    args = %W[
       -DCMAKE_INSTALL_RPATH=#{rpath}
       -DPYTHON=python3
-      -DPYTHON_EXECUTABLE=#{Formula["python@3.9"].opt_bin}/"python3"
+      -DPYTHON_EXECUTABLE=#{Formula["python@3.10"].opt_bin}/"python3"
     ]
 
-    mkdir "macbuild" do
-      system "cmake", *args, ".."
-      system "make"
-      system "make", "install"
-    end
+    system "cmake", "-S", ".", "-B", "macbuild", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
```

Comment:
Should probably be same directory rather than `macbuild` vs. `build`

Reply:
Oops, yes, thanks